### DISCUSSION TITLE
ERROR!

### DISCUSSION THREAD

**Discussion by DHIWGO** [fast4x/RiMusic#2209] [General]
-->English:
ERROR (the application stops): 
Hello friend, the application from version v0.6.37 to v0.6.38 closes the application and does not open it again until the app's data is deleted. This occurs when giving it storage permission so that it can read the music I have it downloaded on my cell phone. 

In version v0.6.36.1, reading the device's storage works correctly and the application does not close.

please fix it, thank you. Watch the video and you will realize what happens. cell phone: Huawei Y8s
Android: 9

-->Italian:
ERRORE (l'applicazione si ferma): 
Ciao amico, l'applicazione dalla versione v0.6.37 alla v0.6.38 chiude l'applicazione e non la riapre finché i dati dell'app non vengono eliminati. Ciò si verifica quando gli si concede il permesso di archiviazione in modo che possa leggere la musica che ho scaricato sul mio cellulare.

Nella versione v0.6.36.1, la lettura della memoria del dispositivo funziona correttamente e l'applicazione non si chiude. 

per favore aggiustalo, grazie. Guardate il video e vi renderete conto di cosa succede. cellulare: Huawei Y8s
Androide: 9
@fast4x 
**VIDEO:↴**
https://github.com/fast4x/RiMusic/assets/163943853/0be773d3-b60a-4183-8759-4e12074d5be7

![IMG_20240607_224738_212.jpg](https://github.com/fast4x/RiMusic/assets/163943853/baf57026-bf69-4ca6-8413-ac1ea9cc45c2)

![IMG_20240607_224717_024.jpg](https://github.com/fast4x/RiMusic/assets/163943853/899e22da-c1c2-4472-a81c-dbcdcc6cb850)

**Comment by DHIWGO** [comment]
> -->English:
> ERROR (the application stops): 
> Hello friend, the application from version v0.6.37 to v0.6.38 closes the application and does not open it again until the app's data is deleted. This occurs when giving it storage permission so that it can read the music I have it downloaded on my cell phone. 
> 
> In version v0.6.36.1, reading the device's storage works correctly and the application does not close.
> 
> please fix it, thank you. Watch the video and you will realize what happens. cell phone: Huawei Y8s
> Android: 9
> 
> -->Italian:
> ERRORE (l'applicazione si ferma): 
> Ciao amico, l'applicazione dalla versione v0.6.37 alla v0.6.38 chiude l'applicazione e non la riapre finché i dati dell'app non vengono eliminati. Ciò si verifica quando gli si concede il permesso di archiviazione in modo che possa leggere la musica che ho scaricato sul mio cellulare.
> 
> Nella versione v0.6.36.1, la lettura della memoria del dispositivo funziona correttamente e l'applicazione non si chiude. 
> 
> per favore aggiustalo, grazie. Guardate il video e vi renderete conto di cosa succede. cellulare: Huawei Y8s
> Androide: 9
> @fast4x 
> **VIDEO:↴**
> https://github.com/fast4x/RiMusic/assets/163943853/0be773d3-b60a-4183-8759-4e12074d5be7
> 
> ![IMG_20240607_224738_212.jpg](https://github.com/fast4x/RiMusic/assets/163943853/baf57026-bf69-4ca6-8413-ac1ea9cc45c2)
> 
> ![IMG_20240607_224717_024.jpg](https://github.com/fast4x/RiMusic/assets/163943853/899e22da-c1c2-4472-a81c-dbcdcc6cb850)
> 
> 
>

**Reply by fast4x** [reply]
ok I'll check it

**Reply by DHIWGO** [reply]
![Screenshot_20240608_012321_it.fast4x.rimusic.jpg](https://github.com/fast4x/RiMusic/assets/163943853/5645cbba-2b96-4e05-84bf-a360e16182bd)

<3
@fast4x

**Reply by fast4x** [reply]
Fixed I'm preparing custom version

**Reply by fast4x** [reply]
Download and try custom version https://github.com/fast4x/RiMusic/tree/master/customVersion

**Reply by DHIWGO** [reply]
Hi
you already solved it, it works.<3

CIAO
hai già risolto, funziona.<3
**VIDEO:↴**
https://github.com/fast4x/RiMusic/assets/163943853/c486e9f7-1b9c-47f4-a30a-e3b4ae7f4846

@fast4x

**Reply by DHIWGO** [reply]
If I find another error I'll let you know brother.Thank you for your dedication and work. :)

Se trovo un altro errore ti farò sapere fratello. Grazie per la vostra dedizione e il vostro lavoro. :)

@fast4x

**Reply by fast4x** [reply]
Thanks
